custom-steps/ffmpeg/pre-build.ps1

```diff
@@ -0,0 +1,12 @@
+Import-Module "$PSScriptRoot/../../ps-modules/Util"
+
+$IsOnMacOS = Get-IsOnMacOS
+
+if(-not $IsOnMacOS) {
+    Write-Host "The step is only required for MacOS.  Skipping step..."
```

Comment:
I think we can get rid of all of these messages

Reply:
Fixed: e5d93971c0d8e3fb07bea1f99163f42b6d52d2e1